<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SNR</class>
 <widget class="QMainWindow" name="SNR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1142</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LSL Visualization</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>370</y>
      <width>671</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="videoLayout"/>
   </widget>
   <widget class="QLabel" name="logoHolder">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>671</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="queryLayout"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>70</y>
      <width>271</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="imageLayout"/>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>360</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>440</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="manualControlButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>520</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Manual Control (ON/OFF)</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSNR">
   <property name="text">
    <string>SNR</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
